fulp_modules/departmental_ert/code/ert_outfit.dm

```diff
@@ -0,0 +1,157 @@
+//
+// ERT Outfits
+//
+
+// MEDICAL ERT OUTFITS
+
+/datum/outfit/centcom/ert/medic/specialized
+	name = "ERT Medic - Specialized"
+
+	belt = /obj/item/storage/belt/medical/advanced
+	gloves = /obj/item/clothing/gloves/combat/nitrile
+
+/datum/outfit/centcom/ert/commander/medical
+	name = "ERT Commander - Medical"
+
+	belt = /obj/item/storage/belt/medical/advanced
+	suit = /obj/item/clothing/suit/space/hardsuit/ert/commandermed
+	back = /obj/item/storage/backpack/ert/medical
+	glasses = /obj/item/clothing/glasses/hud/health/sunglasses
+	gloves = /obj/item/clothing/gloves/combat/nitrile
+	suit_store = /obj/item/gun/energy/e_gun
+	backpack_contents = list(/obj/item/storage/box/survival/engineer=1,\
+		/obj/item/melee/baton/loaded=1,\
+		/obj/item/reagent_containers/hypospray/combat/nanites=1,\
+		/obj/item/gun/medbeam=1)
+
+/datum/outfit/centcom/ert/medic/specialized/alien
+	name = "ERT Medic - Alien"
+
+	uniform = /obj/item/clothing/under/abductor
+	belt = /obj/item/storage/belt/medical/alien
+	backpack_contents = list(/obj/item/storage/box/survival/engineer=1,\
+		/obj/item/gun/energy/alien = 1,\
+		/obj/item/melee/baton/loaded=1,\
+		/obj/item/reagent_containers/hypospray/combat=1,\
+		/obj/item/gun/medbeam=1)
+
+/datum/outfit/centcom/ert/medic/specialized/oath
+	name = "ERT Medic - Oath"
+
+	r_hand = /obj/item/rod_of_asclepius
```

Comment:
```suggestion
	l_hand = /obj/item/rod_of_asclepius
```
It will override the medkit but the r_hand is where plasmaman internals spawn and is mostly left reserved for them.